iocore/net/QUICNetVConnection.cc

```diff
@@ -1525,20 +1527,19 @@ QUICNetVConnection::_packetize_frames(QUICEncryptionLevel level, uint64_t max_pa
   first_block->alloc(iobuffer_size_to_index(0));
   first_block->fill(0);
 
-  if (!this->_has_ack_eliciting_packet_out) {
-    // Sent too much ack_only packet. At this moment we need to packetize a ping frame
-    // to force peer send ack frame.
-    this->_pinger->request(level);
```

Comment:
Assume we send an ack-only packet out. So _has_ack_eliciting_packet_out will be false. The pinger will be triggered for even though we have nothing to send . It will fall into infinite loop when we are using traffic_quic.